modules/twinklearv.js

```diff
@@ -651,10 +651,9 @@ Twinkle.arv.callback.evaluate = function(e) {
 
 			var puppetReport = form.category.value === "puppet";
 			if (puppetReport && !(form.sockmaster.value.trim())) {
-				if (!confirm("You have not entered a sockmaster account for this puppet. Do you want to report this account as a sockpuppeteer instead?")) {
-					return;
-				}
-				puppetReport = false;
+				alert("You have not entered a sockmaster account for this puppet. Consider reporting this account as a sockpuppeteer instead.");
+			} else if (!puppetReport && !(form.sockpuppet[0].value.trim())) {
+				alert("You have not entered any sockpuppet account(s) for this sockmaster. Consider reporting this account as a sockpuppet instead.");
```

Comment:
Shouldn't we `return` after these alerts?

Reply:
Ugh yes, absolutely: frequently when testing I insert `return`s to slow things down, etc.  Got in the habit of removing them all!  Fixed.